deployments/datahub/images/default/requirements.txt

```diff
@@ -57,8 +57,8 @@ astropy==5.1
 dustmaps==1.0.9
 george==0.4.0
 exoplanet==0.5.2
-torch==1.12.1
-torchvision==0.13.1
+torch==1.13.1
```

Comment:
hey @balajialg is this course still using datahub?

`astr 128/256; spring 2021`

Reply:
@shaneknapp Seems like this course is not currently offered this semester - https://classes.berkeley.edu/search/class/astronomy%20128?retain-filters=1&f%5B0%5D=im_field_term_name%3A2729